src/PluginsRegistry.js

```diff
@@ -6,46 +6,59 @@
  * @api public
  */
 export default class PluginsRegistry {
-  registry: Array<Function>;
+  registry: Array<Object>
+  jss: Object
 
-  constructor(): void {
+  constructor(jss: Object): void {
     this.registry = []
+    this.jss = jss
```

Comment:
I think that this should be passed into an instance method instead of storing the reference. The plugin registry doesn't need to retain knowledge of the whole `jss` instance IMO.